randomActivityGen.py

```diff
@@ -23,6 +23,8 @@
     --schools.begin-age=args    The range of ages at which students start going to school [default: 6,20]
     --schools.end-age=args      The range of ages at which students stops going to school [default: 10,30]
     --schools.capacity=args     The range for capacity in schools [default: 100,500]
+    --bus-stop.distance N       Minimum distance between bus stops [default: 10]
```

Comment:
SUMO's units are meters, so this default value means there will be bus stops every 10-20 meters 🤔 
I think something like ~300 is more fitting

Reply:
Changed in 5d6b58d, 10 was just a temporary value I forgot to change